packages/scroller/src/vaadin-scroller.js

```diff
@@ -85,6 +85,30 @@ class Scroller extends FocusMixin(ElementMixin(ThemableMixin(PolymerElement))) {
     };
   }
 
+  /** @protected */
+  ready() {
+    super.ready();
+
+    // Update overflow attribute on resize
+    this.__resizeObserver = new ResizeObserver(() => {
+      this.__updateOverflow();
+    });
+    this.__resizeObserver.observe(this);
```

Comment:
To me this more robust, we should actually observe all the child elements of the scroller.
```suggestion
    this.__resizeObserver.observe(this);
    for (let i = 0; i < this.children.length; i++) {
      this.__resizeObserver.observe(this.children[i]);
    })
```

Reply:
Yes, this is how it works in `vaadin-tabs`.